### ISSUE TITLE
parallel build fails

### ISSUE THREAD

**Issue by kdudka** [cockpit-project/cockpit#2505]
The project fails to build in parallel.  Minimal example:
```
./autogen.sh
./configure
make -j
```

It ends up like this on HP Z400 running Fedora 22:
```
[...]
  AR       libretest.a
ar: `u' modifier ignored since `D' is the default (see `U')
gzip: pkg/base1/require.min.js: No such file or directory
Makefile:8359: recipe for target 'pkg/base1/require.min.js.gz' failed
make[2]: *** [pkg/base1/require.min.js.gz] Error 1
make[2]: *** Waiting for unfinished jobs....
ar: `u' modifier ignored since `D' is the default (see `U')
  GZIP     po/po.da.js.gz
  GZIP     po/po.de.js.gz
  GZIP     po/po.pl.js.gz
  CCLD     mock-sshd
  GZIP     pkg/base1/jquery.min.js.gz
gzip: pkg/base1/jquery.min.js: No such file or directory
Makefile:8359: recipe for target 'pkg/base1/jquery.min.js.gz' failed
make[2]: *** [pkg/base1/jquery.min.js.gz] Error 1
  GZIP     pkg/base1/mustache.min.js.gz
gzip: pkg/base1/mustache.min.js: No such file or directory
Makefile:8359: recipe for target 'pkg/base1/mustache.min.js.gz' failed
make[2]: *** [pkg/base1/mustache.min.js.gz] Error 1
```

**Comment by stefwalter** [comment]
That's very strange. the file pkg/base1/jquery.min.js always exists. I'm unable to reproduce here. Maybe I don't have enough CPUs? I need help reproducing this, or more info.

**Comment by kdudka** [comment]
Indeed.  The file gets removed by ```make clean```, along with several other files.  Files tracked by git should IMHO not be removed by ```make clean```.

Try this:
```
$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working directory clean

$ ./autogen.sh 
[...]

$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
nothing to commit, working directory clean

$ make clean 
[...]

$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

        deleted:    pkg/base1/jquery.min.js
        deleted:    pkg/base1/mustache.min.js
        deleted:    pkg/base1/require.min.js
        deleted:    pkg/kubernetes/d3.min.js
        deleted:    pkg/systemd/journal_day_header.mustache
        deleted:    pkg/systemd/journal_line.mustache
        deleted:    pkg/systemd/journal_reboot.mustache
        deleted:    src/bridge/mock-resource/home/cockpit/another/test-file.min.txt.gz
        deleted:    src/common/mock-content/large.min.js.gz
        deleted:    src/common/mock-content/test-file.txt.gz

no changes added to commit (use "git add" and/or "git commit -a")
```

Anyway, this seems to be an orthogonal issue to the parallel build failure.

**Comment by stefwalter** [comment]
Most people who work on cockpit don't use the same srcdir as builddir and that may be the source of the issue. Let me see.

**Comment by kdudka** [comment]
Thanks!  Both the issues appear fixed in da8e3382.  I would be great if the parallel builds were re-enabled in Fedora at some point.
